doc/source/namespaces.rst

```diff
@@ -26,27 +26,27 @@ Named actors are only accessible within their namespaces.
       pass
 
     # Job 1 creates two actors, "orange" and "purple" in the "colors" namespace.
-    with ray.client().namespace("colors").connect():
+    with ray.init("local://", namespace="colors"):
```

Comment:
didn't we discuss removing "local://"? what is the difference over `ray.init(namespace="colors")`?

Reply:
Will drop from here, main difference right now is that you can't use `with` statements with regular ray.init